x-pack/test_serverless/shared/config.base.ts

```diff
@@ -75,6 +75,7 @@ export default async () => {
         `--server.restrictInternalApis=true`,
         `--server.port=${servers.kibana.port}`,
         '--status.allowAnonymous=true',
+        `--migrations.zdt.runOnRoles=${JSON.stringify(['ui'])}`,
```

Comment:
Should we remove it as soon as #162673 is merged? If so, could you add a comment